pkg/server/endpoints/node/handler.go

```diff
@@ -512,7 +512,7 @@ func (h *Handler) updateNodeSelectors(ctx context.Context,
 	var selectors []*common.Selector
 	if nodeResolver == nil {
 		// If not matching node resolver found, skip adding additional selectors
-		h.c.Log.Debug("could not find node resolver type %q", attestationType)
+		h.c.Log.Debugf("could not find node resolver type %v", attestationType)
```

Comment:
nit: Can we keep the ÷q format specifier? It adds quotes around the attestor type that helps provide visual distinction.

Reply:
Sorry, fixed in the amended commit...